src/virtualship/instruments/xbt.py

```diff
@@ -0,0 +1,137 @@
+"""XBT instrument."""
+
+from dataclasses import dataclass
+from datetime import timedelta
+from pathlib import Path
+
+import numpy as np
+from parcels import FieldSet, JITParticle, ParticleSet, Variable
+
+from ..spacetime import Spacetime
+
+
+@dataclass
+class XBT:
+    """Configuration for a single XBT."""
+
+    spacetime: Spacetime
+    min_depth: float
+    max_depth: float
+    initial_fall_speed: float
+    deceleration_coefficient: float
+
+
+_XBTParticle = JITParticle.add_variables(
+    [
+        Variable("temperature", dtype=np.float32, initial=np.nan),
+        Variable("max_depth", dtype=np.float32),
+        Variable("min_depth", dtype=np.float32),
+        Variable("fall_speed", dtype=np.float32),
+        Variable("initial_fall_speed", dtype=np.float32),
```

Comment:
Why does this need to be a separate variable? Can't we just set `fall_speed` above to the initial version at the beginning? Why store it as a separate `Variable`?